apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/KeyStoreController.java

```diff
@@ -219,16 +215,17 @@ public Response downloadKeyStore(@FormParam("account") String account,
                 account2FAService.verify2FA(twoFactorAuthParameters);
```

Comment:
It's a good idea to use Secured2FA annotation for this endpoint instead of using the verify2FA() service method.